### DISCUSSION TITLE
Error: Command "yarn install" exited with 1

### DISCUSSION THREAD

**Discussion by jainpawan21** [vercel/community#469] [Help] [answered]
During every new deployment, I get this error, and deployments get failed. It seems that this is a cache issue.

<img width="1012" alt="Screenshot 2022-03-14 at 8 03 20 PM" src="https://user-images.githubusercontent.com/39362422/158195650-7fd347ef-5c7f-4139-b2e4-ee0ad642a3d5.png">

When I click on redeploy then the deployment is successful. How to solve this issue?

**Comment by swarnava** [comment, marked as answer]
Hi, Are you using Yarn 2? We don't fully support that yet: https://vercel.com/support/articles/does-vercel-support-yarn-2.
